mod-users-bl/mod-users-bl.postman_collection.json

```diff
@@ -2134,25 +2133,26 @@
 									"    tv4.addSchema(\"proxyFor\", JSON.parse(JSON.stringify(pm.variables.get(\"mod-users/proxyfor.json\"))));",
 									"    tv4.addSchema(\"metadata.schema\", JSON.parse(JSON.stringify(pm.variables.get(\"metadata.schema\"))));",
 									"",
-									"    pm.expect(tv4.validate(pm.response.json(), JSON.parse(JSON.stringify(pm.variables.get(\"compositeUser.json\"))))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+									"    pm.expect(tv4.validate(pm.response.json(), JSON.parse(JSON.stringify(pm.variables.get(\"compositeUser.json\"))))).to.equal(false, \"Schema validation error: \" + JSON.stringify(tv4.error));",
```

Comment:
This is a bit confusing. Can you explain the need for this change?

Reply:
Updated it Matt. Thanks for your feedback!